include/deal.II/grid/tria_iterator.h

```diff
@@ -165,10 +165,10 @@ class TriaActiveIterator;
  *
  * The accessors provided by the library consists of two groups, determined by
```

Comment:
while you're here, can you replace "consists" by "consist" to get the grammar right? Though maybe "fall" or "can be grouped into two classes" would actually be better